contrib/collection.cgi

```diff
@@ -809,6 +834,7 @@ sub action_show_graph
 
     $file =~ s/:/\\:/g;
     s/{file}/$file/ for (@rrd_args);
+    s/{ds(\d+)}/$Types->{$gtype}[$1]/ for (@rrd_args);
```

Comment:
I think `for` sets `$_` instead of `$1`

Reply:
Yes, the `$1` here is referring to the `(\d+)` capture group in the regex.  The `for` is to do the ds(\d+) replacement for each item in @rrd_args.

Reply:
Ah, ok, missed that.